### PR TITLE
docs: add instructions on how to deploy to google cloud run

### PR DESCRIPTION
**Pull request by udit-001**

### CODE REVIEW

**Comment by udit-001** [comment]
Tagging @Manouchehri & @marcklingen for review.

File: pages/docs/deployment/self-host.mdx [hunk 1]
```diff
@@ -211,6 +211,41 @@ Optionally, you can apply the new model definitions to existing data using the f
...
+11. Configure ingress control according to your needs. For most cases, 'All' should suffice.
+12. Under "Container(s), Volumes, Networking & Security", specify container port as `3000`. Then click the "Secrets & Variables" tab. Add the required environment variables: `SALT`, `NEXTAUTH_URL`, `NEXTAUTH_SECRET`, and `DATABASE_URL`, etc.
+13. The `NEXTAUTH_URL` will be the URL of the deployed service, which you can update later when the service finishes building and gets deployed.
+14. Now, set up the database. Note: Your Cloud Run service won't be assigned a static IP, so you can't whitelist the ingress IP in Cloud SQL or any other hosted databases. Instead, utilize the Google Cloud Proxy. Scroll down to "Cloud SQL connections" and enable access to the SQL instance hosting the database for Langfuse. After that, go to the "Secrets & Variables" section of the Container setup and add `DATABASE_URL` as an environment variable.
+
+    Here's what the value should look like:
+
+    ```
+    postgresql://<user-name>:<password>@localhost/<db-name>/?host=/cloudsql/<google-cloud-project-id>:<region-id>:<sql-instance-id>&sslmode=none&pgbouncer=true
+    ```
```
Comment (Manouchehri) [inline pages/docs/deployment/self-host.mdx:243]: You should add `DIRECT_URL` as well, but without the `&pgbouncer=true`.

Comment (udit-001) [reply pages/docs/deployment/self-host.mdx:243]: This is done! 👍🏼

**Comment by marcklingen** [comment]
Thanks for the contribution, looks good to me and really helpful for everyone deploying to Cloud Run for the first time.

I made a tiny edit as the `main` branch is used for staging and includes pre-release changes.

@udit-001 please accept the [cla](https://cla-assistant.io/langfuse/langfuse-docs?pullRequest=456)

**Comment by marcklingen** [comment]
@Manouchehri appreciate your review! thank you

**Comment by udit-001** [comment]
@marcklingen should I delete the old passage at the start which is now redundant?

> Cloud Run does not support deploying containers directly from the GitHub Container Registry (ghcr.io) used by Langfuse. To avoid the need to pull the container and push it to the Google registry, you can refer to this guide or use Cloud Build. Note: you can use Google for SSO.

**Comment by marcklingen** [comment]
> @marcklingen should I delete the old passage at the start which is now redundant?
> 
> > Cloud Run does not support deploying containers directly from the GitHub Container Registry (ghcr.io) used by Langfuse. To avoid the need to pull the container and push it to the Google registry, you can refer to this guide or use Cloud Build. Note: you can use Google for SSO.

makes sense, thank you!

**Comment by marcklingen** [comment]
thanks @udit-001! I'll merge once you accepted the [CLA](https://cla-assistant.io/langfuse/langfuse-docs?pullRequest=456)

**Comment by marcklingen** [comment]
made minor final edits to make clear that the `production` branch needs to be used, merging now, thank you @udit-001 @Manouchehri!

**Pull request merged**